docs/updating/from_4.0/to_4.1.md

```diff
@@ -0,0 +1,251 @@
+---
+latest_tag: '4.1.1'
+---
+
+# Update from v4.0.x to v4.1
+
+This update procedure applies if you are using v4.0.latest.
+
+Go through the following steps to update to v4.1.
+
+## Update the app to v4.1
+
+First, run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer require ibexa/experience:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer require ibexa/commerce:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+Continue with updating the app:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer recipes:install ibexa/content --force -v
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer recipes:install ibexa/experience --force -v
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer recipes:install ibexa/commerce --force -v
+    ```
+
+The `recipes:install` command installs new YAML configuration files. Look through the old YAML files and move your custom configuration to the relevant new files.
+
+Review the `bundles.php` and leave only third-party entires and entries added by the `recipes:install` command, that start with `Ibexa\Bundle`.
+
+## Update the database
+
+Apply the following database update script:
```

Comment:
This script is contained in https://github.com/ibexa/installer/blob/main/upgrade/db/mysql/ibexa-4.0.0-to-4.1.0.sql. Any reason not to ask the user to apply the existing one?

Reply:
This script is available only in the DXP (and is needed only by the DXP - it does not contain OSS related changes).

Maybe, to avoid confusion for OSS, we mention it somehow?
Similar as in https://github.com/ezsystems/developer-documentation/pull/1595/files#diff-6e55313fbe64f4322272daed3c5422857200c6ecd942f4173848cd6ad02046cfR118-R130 - if no access to ibexa/installer then do nothing?